### PR TITLE
London 10 - Elena Barker - Full - Stack - Project - Assessment - Level 100 - Week 1

### PR DESCRIPTION
**Pull request by ElenaBarker**

### CODE REVIEW

**Comment by ShayanMahnam** [comment]
Hello Hello, Please put the node_module on .gitignore so i can review your changes

**Comment by ElenaBarker** [comment]
Done )

File: client/src/App.js [hunk 1]
```diff
@@ -1,13 +1,37 @@
...
+const App = () => {
+  const [videos, setVideos] = useState(videosData);
+
+  const handleUpVote = (video) => {};
+
+  const handleDownVote = (video) => {};
+
+  const handleRemoveVideo = (video) => {};
+
+  const handleAddVideo = (newVideo) => {};
```
Comment (sherif98) [inline client/src/App.js:16]: Don't you need to implement those functions for level 100?

**Review by sherif98 — CHANGES_REQUESTED** [review]

File: client/.gitignore
[the remark this answers is not in the document]
Comment (ElenaBarker) [reply client/.gitignore:1]: I deleted .gitignore from client, thank you.

File: client/src/App.js (hunk 1 above)
Comment (ElenaBarker) [reply client/src/App.js:16]: Sorry, my fault, will do that (

**Pull request closed without merging**